### PR TITLE
Handle GEOMETRY columns with include option

### PR DESCRIPTION
**Pull request by alxarch**
#4715

### CODE REVIEW

File: lib/dialects/mysql/query-generator.js [hunk 1]
```diff
@@ -353,7 +353,11 @@ var QueryGenerator = {
   },
 
   geometrySelect: function(column) {
-    return 'AsText(' + this.quoteIdentifiers(column) + ') AS ' + column;
+    return {
+      as: column,
+      fn: 'AsText',
+      cols: [column]
+    };
```
Comment (mickhansen) [inline lib/dialects/mysql/query-generator.js]: This looks like completely new syntax?
Would `return [sequelize.fn('AsText', sequelize.col(column)), column]` not accomplish the same thing?

Comment (alxarch) [inline lib/dialects/mysql/query-generator.js]: Where would i get the Sequelize instance inside the abstract query generator?

**Comment by alxarch** [comment]
This is a temporary fix. As it seems that the issue will be addressed in a different manner by #4186

**Comment by janmeier** [comment]
Handled in https://github.com/sequelize/sequelize/issues/4834

**Pull request closed without merging**